wrappers/rust/icicle-core/src/polynomials/mod.rs

```diff
@@ -0,0 +1,707 @@
+#[macro_export]
+macro_rules! impl_univariate_polynomial_api {
+    (
+        $field_prefix:literal,
+        $field_prefix_ident:ident,
+        $field:ident
+    ) => {
+        use icicle_core::traits::FieldImpl;
+        use icicle_cuda_runtime::memory::HostOrDeviceSlice;
+        use std::clone;
+        use std::cmp;
+        use std::ffi::c_void;
+        use std::ops::{Add, AddAssign, Div, Mul, Rem, Sub};
```

Comment:
I can't find a standard that we have for grouping mods from the same crate but I'd prefer a single `use` per crate unless its necessary to break a mod out for conditional inclusion.

```suggestion
        use std::{
            clone, cmp, 
            ffi::c_void,
            ops::{Add, AddAssign, Div, Mul, Rem, Sub}
        };
```

Reply:
fixed